lib/material/extra/input_chips/material_input_chips.flow

```diff
@@ -3,25 +3,31 @@ import material/material_ui;
 export {
 	MInputChips(items : Transform<[string]>, style : [MInputChipsStyle]) -> Material;
 
-	MInputChipsStyle ::= MInputChipsMaxWidth, MSelectedInputChips, MInputChipsLeftCaption, MCustomAutoCompleteStyle, MCustomChipStyle,
-		MInputChipsMultiline, MAnimationEnabled /*for multiline*/;
+	MInputChipsStyle ::= MInputChipsMaxWidth, MSelectedInputChips, MInputChipsLeftCaption, MCustomAutoCompleteStyle, MCustomChipStyle, MInputChipsTextButton,
+		MInputChipsToRight, MInputChipsInputTrigger, MInputChipsMultiline, MAnimationEnabled /*for multiline*/;
 
 		MSelectedInputChips(items : DynamicBehaviour<[string]>);
 		MInputChipsLeftCaption(caption : Material);
+		MInputChipsTextButton(text : string);
+		MInputChipsToRight(left : bool);
 		MInputChipsMaxWidth(width : Transform<double>);
 		MCustomChipStyle(style : [MChipStyle]);
 		MCustomAutoCompleteStyle(style : [MAutoCompleteStyle]);
 		MInputChipsMultiline();
+		MInputChipsInputTrigger(trigget : DynamicBehaviour<bool>);
```

Comment:
'trigget' typo

Reply:
Also, don't understand how is it meant to be used.

Reply:
Fixed to MInputChipsText(text : Dynamic Behaviour<string>);
to keep track of whether the string is empty